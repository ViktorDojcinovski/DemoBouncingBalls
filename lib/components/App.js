import React from 'react';
import styled from 'styled-components';

import BouncingBallsEnv from './BouncingBallsEnv';

const StyledContainer = styled.div`
  width: 100%;
  height: 100%;
  margin: 0;
  padding: 0;
  overflow: hidden;
  background-color: #17191a;

  .bouncy {
    display: inline-block;
    width: 20px;
    height: 20px;
    border-radius: 50%;
    background-color: greenyellow;
    position: absolute;
  }
`;

export default function App() {
  return (
    <StyledContainer>
      <BouncingBallsEnv />
    </StyledContainer>
  );
}
