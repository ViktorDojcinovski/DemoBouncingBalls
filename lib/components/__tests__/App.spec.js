import React from 'react';
import renderer from 'react-test-renderer';

import App from '../App';

describe('App', () => {
  it('should render properly', () => {
    //...arrange
    const component = renderer.create(<App />);
    //...act
    //...assert
    expect(component).toMatchSnapshot();
  });
});
