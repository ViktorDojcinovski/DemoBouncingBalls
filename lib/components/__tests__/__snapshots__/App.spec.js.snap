// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render properly 1`] = `
<div
  className="sc-bwzfXH fKwfZK"
>
  <section
    className="sc-bdVaJa esgPqa"
  >
    <div
      className="wrapper"
      onClick={[Function]}
    />
  </section>
</div>
`;
